### ISSUE TITLE
Documentation typo?

### ISSUE THREAD

**Issue by NicoHood** [gtalarico/pyairtable#200]
Look at this:
https://pyairtable.readthedocs.io/en/latest/orm.html

![grafik](https://user-images.githubusercontent.com/6888294/197544078-7c1d94f1-94b7-4f3f-970c-7bbd044dc6b4.png)

It says:
```py
partner = fields.LinkField("Partner", "Contact", lazy=False)
```

The second parameter is a string. In the code itself this is documented as object:
https://github.com/gtalarico/pyairtable/blob/main/pyairtable/orm/fields.py#L43

I guess that must be corrected in one of the two.

**Comment by gtalarico** [comment]
Good catch.

Doc should show model instance not `str`.
I intended to support both but have not implemented. 
The `str` option allows circular referencing (django and my support similar approaches where it loads object by name reference)

![image](https://user-images.githubusercontent.com/9513968/197590021-e6834a06-3106-4c6b-accf-852162ed7145.png)

**Comment by mesozoic** [comment]
This was resolved in #277; see https://pyairtable.readthedocs.io/en/stable/orm.html#linked-records
